contributing/code/bc.rst

```diff
@@ -277,6 +277,12 @@ Change value of a constant                          Yes [1]_ [5]_
        Additionally, if a constant will likely be used in objects that are
        serialized, the value of a constant should not be changed.
 
+.. [6] Allowed using the ``@final`` annotation.
+
+.. [7] Allowed if the class is final.
```

Comment:
or when the class has the `@final` annotation since the previous major version (same below)